src/components/InjectedComponents/TypedMessageRenderer.tsx

```diff
@@ -54,13 +57,32 @@ registerTypedMessageRenderer('text', {
     priority: 0,
 })
 
+export const DefaultTypedMessageAnchorRenderer = React.memo(function DefaultTypedMessageAnchorRenderer(
+    props: TypedMessageRendererProps<TypedMessageAnchor>,
+) {
+    const { content, href } = props.message
+    return renderWithMetadata(
+        props,
+        <Typography component="span" variant="body1" data-testid="anchor_payload">
```

Comment:
Does it necessary to wrap a `Link` in `Typography`?

Reply:
Easier to handle the style of the link.